src/bn256/fq2.rs

```diff
@@ -271,12 +272,14 @@ impl Fq2 {
     }
 
     pub fn frobenius_map(&mut self, power: usize) {
-        self.c1 *= &FROBENIUS_COEFF_FQ2_C1[power % 2];
+        if power % 2 != 0 {
+            self.conjugate()
+        }
```

Comment:
Can't you shift by one to the right and && by a mask? Might be harder to read but we're sure that will always be as optimized as possible.

Reply:
Not sure how to do this properly since `power` is `usize`.
We can use its methods for example:
```  rust
if power.trailing_zeros() != 0 {
    self.conjugate()
        }
```
but I don't think it is worth it.